src/driver/main.cpp

```diff
@@ -483,6 +487,23 @@ struct perf : command<perf>
     }
 };
 
+struct mark : command<mark>
```

Comment:
Since this is for roctx, maybe the command should be `roctx`?

Reply:
Changed it.